### PR TITLE
Lint & Format Proposal

### PR DESCRIPTION
**Pull request by gaelreyrol**
Hi,

Following the previous PR I made #78, I choosed to open a new one to allow a fresh & clean review.
Thanks @alexisjanvier @zyhou for your feedback.

I did not included `env` file changes and `commitlint` suggestion because I thought it was out of scope.

Hope this time it will do the job !

### CODE REVIEW

**Review by zyhou — APPROVED** [review]
Clean nice job :+1:

File: .editorconfig [hunk 1]
```diff
@@ -1,11 +1,12 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
```
Comment (alexisjanvier) [inline .editorconfig:4]: Crap... there's gonna be a debate. I confess I'm more `space` than `tab`.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+	"useTabs": true,
```
Comment (alexisjanvier) [inline .prettierrc:2]: same ...

**Review by alexisjanvier — COMMENTED** [review]
That's great :+1: . We just need to settle the `space` vs.` tab` issue.

File: .editorconfig (hunk 1 above)
Comment (gaelreyrol) [reply .editorconfig:4]: I am too (and with 2 spaces) but a lot of code has been written with tabs ^^

**Review by alexisjanvier — APPROVED** [review]
Ok. On review et on merge la PR sur les livraisons pour lancer un déploiement. Et je merge cette PR pour faire un tour complet des fix à appliquer pour être *linter proof** ... (peut-être que je changerais les tab en space à ce moment là ^^)

**Pull request merged**